mlflow/store/tracking/file_store.py

```diff
@@ -734,12 +734,12 @@ def _get_all_metrics(self, run_info):
         return metrics
 
     @staticmethod
-    def _get_metric_from_line(metric_name, metric_line):
+    def _get_metric_from_line(metric_name, metric_line, parent_path):
         metric_parts = metric_line.strip().split(" ")
         if len(metric_parts) != 2 and len(metric_parts) != 3:
             raise MlflowException(
                 "Metric '%s' is malformed; persisted metric data contained %s "
-                "fields. Expected 2 or 3 fields." % (metric_name, len(metric_parts)),
+                "fields. Expected 2 or 3 fields. Experiment path: %s" % (metric_name, len(metric_parts), parent_path),
```

Comment:
Q: Shall we log the experiment name / experiment ID instead of the path (to make it more readable)

Reply:
can we add a test ?

Reply:
+1. @rangit3 happy to merge this once we convert the path to an ID :D